src/render/views/CreateEncounterView.tsx

```diff
@@ -22,6 +29,15 @@ const useStyles = makeStyles(theme =>
         },
         notLastButton: {
             marginRight: theme.spacing(1)
+        },
```

Comment:
Looks like most of these styles aren't being used anymore; can probably remove them